server/pkg/item/item.go

```diff
@@ -36,3 +39,25 @@ func (i *Item) UpdateFields(fields []*Field) {
 	}
 	i.fields = slices.Clone(fields)
 }
+
+func (i *Item) Filtered(list id.FieldIDList) *Item {
```

Comment:
```suggestion
func (i *Item) FilterFields(list id.FieldIDList) *Item {
```

More specifically, indicate what to filter.